lib/blocs/companies_cubit/company_state.dart

```diff
@@ -1,19 +1,25 @@
 import 'package:equatable/equatable.dart';
-import 'package:twake/models/company.dart';
+import 'package:twake/models/company/company.dart';
 
 abstract class CompaniesState extends Equatable {
   const CompaniesState();
 }
 
+class InitState extends CompaniesState {
```

Comment:
@RockinPaul can you please provide us with some uniform naming convention for states?
You can use companies_state as an example.

Reply:
how about `Companies_[state-name]_State` ? 
so in this case `CompaniesInitialState`

Reply:
@bobs4462 @manhduc-nguyen Felix Angelov can: https://bloclibrary.dev/#/blocnamingconventions
I often repeat this again and again keeping this page open during my work with Bloc.

Reply:
Alright let's use the conventions from BLoC library creator. Please add the resource to Notion for further reference.

Reply:
Nice, fixed.